src/usr/local/sbin/ppp-linkdown

```diff
@@ -27,7 +27,9 @@ if [ -f /tmp/${IF}up ] && [ -f /conf/${IF}.log ]; then
 	/usr/local/sbin/ppp-log-uptime.sh $seconds ${IF} &
 fi
 
-/etc/rc.kill_states ${IF} ${LOCAL_IP}
+if [ ${IF} !=~ "ppp[0-9]*" ]; then
```

Comment:
`test` (aka `[`) doesn't  support this operator `!=~` so this will not work on FreeBSD's `/bin/sh`